pkg/operator/ceph/nfs/config.go

```diff
@@ -135,8 +142,21 @@ RGW {
         name = "client.` + userID + `";
 }
 
+`
+
+	if version.IsAtLeast(uuidConfigSuportedCephVersion) {
```

Comment:
We shouldn't need this version check downstream. We know the version is the latest ganesha, so we can always append this value.

Reply:
Isn't this generating the new config as part of the upgrade with the new ceph version? We already know the new ceph version is ganesha 6.x, so I don't see the disconnect.

Reply:
For this ganesha issue, does the ceph version matter other than the nfs daemon? I thought it only affected the nfs daemon.

Reply:
Yaa its just for nfs daemon

Reply:
I will again do a deeper look, updated for now